themes/mongodb/src/css/suggestion.css

```diff
@@ -0,0 +1,81 @@
+html {
+  overflow-x: hidden;
+}
+
+.suggestion {
+  background-color: #7bc552;
+  bottom: 30%;
+  font-family: inherit;
+  overflow-x: hidden;
+  padding: 1rem;
+  position: fixed;
+  right: 20px;
+  transform: translateX(120%);
+  transition: transform 300ms;
+  /* transition-delay: 1.5s; */
+  transition-property: all;
+  width: 195px;
+  z-index: 100;
+}
+
+@media (max-width: 775px) {
+  .sphinxsidebar {
+      display: none;
+  }
+}
+
+.is-open {
+  transform: none;
+}
+
+.is-closed {
+  opacity: 0;
+}
+
+.suggestion h1 {
+  color: white;
+  font-size: 2rem;
+  font-weight: bold;
+  margin-top: 0;
+}
+
+.suggestion-card {
```

Comment:
We use `postcss-nested`, so you can use the `&` operator to lexically nest all of the suggestion card rules:: https://github.com/postcss/postcss-nested